projects/storefrontapp-e2e-cypress/cypress/integration/regression/asm/asm.e2e-spec.ts

```diff
@@ -8,6 +8,14 @@ import { getSampleUser } from '../../../sample-data/checkout-flow';
 import { clearAllStorage } from '../../../support/utils/clear-all-storage';
 import * as consent from '../../../helpers/consent-management';
 import { fillShippingAddress } from '../../../helpers/checkout-forms';
+import {
+  waitForCategoryPage,
+  waitForPage,
+} from '../../../helpers/checkout-flow';
+import {
+  navigateToCategory,
+  navigateToHomepage,
+} from '../../../helpers/vendor/cds/merchandising-carousel';
 
```

Comment:
Could we extract `navigateToCategory` and `navigateToHomepage` in a more common helper than CDS (cds being an integration library)

Reply:
I thought you might say that. Consider it done.